features/database_cleaner.feature

```diff
@@ -16,7 +16,7 @@ Feature: Database Cleaner
           When I create 7 more widgets
           Then I should have 9 widgets
       """
-    And I run `rails generate model widget name:string`
+    And I run `bundle exec rails generate model widget name:string`
```

Comment:
The GitHub Actions environment doesn't have Rails installed globally, so we need to execute this via `bundle exec`